### PR TITLE
Fix some MissingDocblockType or InvalidDocblock warnings.

### PR DESCRIPTION
**Pull request by kesselb**
Fix some InvalidDocblock and MissingDocblockType warnings found by psalm.

### CODE REVIEW

File: apps/dav/lib/CardDAV/PhotoCache.php [hunk 1]
```diff
@@ -36,6 +36,7 @@
 use OCP\Files\SimpleFS\ISimpleFolder;
 use OCP\ILogger;
 use Sabre\CardDAV\Card;
+use Sabre\VObject\Parameter;
```
Comment (kesselb) [inline apps/dav/lib/CardDAV/PhotoCache.php:39]: For https://github.com/nextcloud/server/blob/b13aa660c91873437afec36ef6466ef609b7959c/apps/dav/lib/CardDAV/PhotoCache.php#L266

**Review by MorrisJobke — APPROVED** [review]
Nice 👍

**Comment by MorrisJobke** [comment]
> Fix some InvalidDocblock and MissingDocblockType warnings found by psalm.

#21787

**Review by MorrisJobke — APPROVED** [review]
Yes - makes sense 🙈

**Comment by MorrisJobke** [comment]
Please review :)

**Review by ChristophWurst — APPROVED** [review]

File: apps/files_sharing/templates/public.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
-/** @var $l \OCP\IL10N */
-/** @var $_ array */
+/** @var \OCP\IL10N $_ */
```
Comment (MorrisJobke) [inline apps/files_sharing/templates/public.php:2]: Fixed in https://github.com/nextcloud/server/pull/22272/files#diff-28509333ad5b898bf1af427bd3e64113

**Pull request merged**